### ISSUE TITLE
NTR: Butyric acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8630]
New term request Butyric acid high in urine (segal_070726155124): Butanoic; butyrate

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000433
- label: Elevated urinary butyric acid level
- parents: [Elevated urinary carboxylic acid (HP:0040156)](https://hpo.jax.org/app/browse/term/HP:0040156)
- pmid: 
- def: The amount of butyric acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8630
- synonyms:
